crates/api_common/src/lib.rs

```diff
@@ -446,44 +437,42 @@ pub async fn send_verification_email(
   );
   blocking(pool, move |conn| EmailVerification::create(conn, &form)).await??;
 
-  let subject = format!("Verify your email address for {}", settings.hostname);
-  let body = format!(
-    concat!(
-      "Please click the link below to verify your email address ",
-      "for the account @{}@{}. Ignore this email if the account isn't yours.<br><br>",
-      "<a href=\"{}\">Verify your email</a>"
-    ),
-    username, settings.hostname, verify_link
-  );
-  send_email(&subject, new_email, username, &body, settings)?;
+  let lang = get_user_lang(user);
+  let subject = lang.verify_email_subject(&settings.hostname);
+  let body = lang.verify_email_body(&user.person.name, &settings.hostname, verify_link);
+  send_email(&subject, new_email, &user.person.name, &body, settings)?;
 
   Ok(())
 }
 
 pub fn send_email_verification_success(
-  local_user_view: &LocalUserView,
+  user: &LocalUserView,
   settings: &Settings,
 ) -> Result<(), LemmyError> {
-  let email = &local_user_view.local_user.email.to_owned().expect("email");
-  let subject = &format!("Email verified for {}", local_user_view.person.actor_id);
-  let html = "Your email has been verified.";
-  send_email(subject, email, &local_user_view.person.name, html, settings)
+  let email = &user.local_user.email.to_owned().expect("email");
+  let lang = get_user_lang(user);
+  let subject = &lang.email_verified_subject(&user.person.actor_id);
+  let body = &lang.email_verified_body();
+  send_email(subject, email, &user.person.name, body, settings)
+}
+
+pub fn get_user_lang(user: &LocalUserView) -> Lang {
+  let user_lang = LanguageId::new(user.local_user.lang.clone());
```

Comment:
If this doesn't work correctly for things like `pt-br`, we might have to unfortunately create mappings. But lets not worry about that yet.